models/logs.js

```diff
@@ -35,10 +35,24 @@ const fetchLogs = async (query, param) => {
     let call = logsModel.where("type", "==", param);
     Object.keys(query).forEach((key) => {
       // eslint-disable-next-line security/detect-object-injection
-      call = call.where(key, "==", query[key]);
+      if (key !== "limit" && key !== "lastDocId") {
+        call = call.where(key, "==", query[key]);
+      }
     });
 
-    const snapshot = await call.get();
+    const { limit, lastDocId } = query;
+    let lastDoc;
+    const limitDocuments = Number(limit);
+
+    if (lastDocId) {
+      lastDoc = await logsModel.doc(lastDocId).get();
+    }
+
+    const logsSnapshotQuery = call.orderBy("timestamp", "desc").startAfter(lastDoc ?? "");
```

Comment:
Do we fix the order to descending?

Reply:
Ideally we would want to see the last updated logs and hence kept it desc. You want me to make it configurable by taking a query param from frontend?